app/src/main/kotlin/com/wire/android/ui/home/conversationslist/AllConversationScreen.kt

```diff
@@ -24,6 +24,7 @@ fun AllConversationScreen(
     onEditConversation: (ConversationItem) -> Unit,
     onOpenUserProfile: (UserId) -> Unit,
     onScrollPositionChanged: (Int) -> Unit = {},
+    openConversationNotificationsSettings: (ConversationItem) -> Unit,
```

Comment:
maybe to follow the same pattern, we can rename it to
```suggestion
    onOpenConversationNotificationsSettings: (ConversationItem) -> Unit,
```

Reply:
done